Python/optimizer.c

```diff
@@ -757,9 +757,8 @@ translate_bytecode_to_trace(
                                 opcode == SEND_GEN)
                             {
                                 DPRINTF(2, "Bailing due to dynamic target\n");
-                                ADD_TO_TRACE(uop, oparg, 0, target);
-                                ADD_TO_TRACE(_DYNAMIC_EXIT, 0, 0, 0);
-                                goto done;
+                                OPT_STAT_INC(low_confidence);
```

Comment:
Could you add a "dynamic target" reason? This is different to having low confidence in the direction of branches.

Reply:
Sure, we should just remember to remove it once the other PRs to handle generators and underflow land.